### DISCUSSION TITLE
Star-lists need an update to match the beta 'List'

### DISCUSSION THREAD

**Discussion by xulu199705** [community/community#42095] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The 'List' function just satisfy the requests basiclly. The star-lists should be updated to match the new beta, especially the filter.

- **Multi-selecting** is necessary.
- Also, new **list-based rule** of filter is needed.
  - Hide starred-repos which are in List to get a clear view of misc-starred-repos

**Comment by hel147** [comment]
Good
